UnitySetup/UnitySetup.psm1

```diff
@@ -94,13 +93,15 @@ class UnitySetupInstance {
                 @{
                     [UnitySetupComponent]::Documentation = , [io.path]::Combine("$Path", "Documentation");
                     [UnitySetupComponent]::StandardAssets = , [io.path]::Combine("$Path", "Standard Assets");
+                    #[UnitySetupComponent]::Mac_IL2CPP = , [io.path]::Combine("$playbackEnginePath", "???");
                     [UnitySetupComponent]::Windows = , [io.path]::Combine("$playbackEnginePath", "WindowsStandaloneSupport");
-                    [UnitySetupComponent]::Linux = , [io.path]::Combine("$playbackEnginePath", "LinuxStandaloneSupport");
                 }
             }
         }
 
         # Common playback engines:
+        $componentTests[[UnitySetupComponent]::Linux] = , [io.path]::Combine("$playbackEnginePath", "LinuxStandaloneSupport");
+        $componentTests[[UnitySetupComponent]::Mac] = , [io.path]::Combine("$playbackEnginePath", "MacStandaloneSupport");
```

Comment:
I think it was more clear about what we were testing for before. With this change, on a mac, we'd set the component above, then we'd test this directory, which will never exist. Same as on Linux, when support comes. If we prefer performing tests that we know will always fail on some platforms then we should move the Windows test here as well.

Reply:
Will they fail?

It looked like the exact same paths above, which is why I moved it down.

Reply:
Well, it wouldn't throw an error, but on the mac platform, I don't believe the MacStandaloneSupport folder ever exists, etc.